usabilla-react-native/README.md

```diff
@@ -40,13 +43,39 @@ react-native link usabilla-react-native
 To use this Bridge, please make sure you are using XCode 9.1 or above.
 
 ## Features
-The first version of Usabilla for React Native `0.1.0` allows you to :
-- Load and show a Passive Feedback form.
-- Submit the results of the form.
+### Load and show a Passive Feedback form
+In order to load a passive feedback form with the  Usabilla library few methods need to be implemented:
+- `usabilla.loadFeedbackForm("YOUR_FORM_ID_HERE")`
+  - The method performing the call for the form
```

Comment:
Instead of `The method performing ` I'd say `This method performs`.
It was not so clear to me the first time. This goes for all points